services/slam/builtin/builtin.go

```diff
@@ -202,6 +209,7 @@ func runtimeServiceValidation(
 			path, err = slamSvc.getAndSaveDataDense(ctx, cams)
 			paths = append(paths, path)
 		default:
+			// TODO[kat]: Doesn't this mean that the slam AlgoType is not defined?
```

Comment:
[nit] We shouldn't leave TODOs in the code unless they're tied to a particular ticket.